src/pixelator/plot/__init__.py

```diff
@@ -10,7 +10,17 @@
 from matplotlib.colors import Normalize
 import networkx as nx
 import pandas as pd
-from typing import Literal, Tuple
+from typing import Literal, Tuple, Optional
```

Comment:
Let's try to keep lexicographic order here. We sometimes enforce this.

Reply:
Should be fixed now!